### DISCUSSION TITLE
翻訳作業のバージョンについて

### DISCUSSION THREAD

**Discussion by denev-rove** [nsk4762jp/OxygenNotIncluded-Japanese#252] [General]
今後の作業方針についての相談です。

今回、ホットキーのショートカットを表示する文言がローカライズされない現象が発生しましたが、その原因は、翻訳ファイルの仕様が変更されたためでした。

おそらくこの翻訳プロジェクトは、テストバージョンではなく、製品バージョン（公開バージョン）で作業されているかと思います。
そのため、今回のような仕様の変更を、製品バージョンの公開まで気づくことができないワークフローになっています。

そこで、この翻訳プロジェクトを、テストバージョンに基づくワークフローに変更できないでしょうか。
テストバージョンと製品バージョンは基本的に互換性は保証されません（たいていは問題ないですが、今回のような仕様変更もあります）が、Steam Workshopへの公開を製品バージョンの公開に合わせるようにすれば、この翻訳MODの一般利用者に影響はないはずです。
（テストバージョンで遊んでいる人は、このサイトからstrings.poをダウンロードすることもできます）

ご検討をお願いします。

**Comment by nsk4762jp** [comment]
仰ることは概ね同意なんですが、体力と時間の都合でテストブランチへの追従は(僕自身では)やらないつもりです。

公開版が出たときにどのみち修正作業が発生することを考えると、テストブランチを積極的に追従する理由は特にないかなぁと思っています。

*.po ファイルをテストブランチに対応させること自体は簡単です。
[[備忘録] このリポジトリの運用方法](https://github.com/nsk4762jp/OxygenNotIncluded-Japanese#%E5%82%99%E5%BF%98%E9%8C%B2-%E3%81%93%E3%81%AE%E3%83%AA%E3%83%9D%E3%82%B8%E3%83%88%E3%83%AA%E3%81%AE%E9%81%8B%E7%94%A8%E6%96%B9%E6%B3%95) にある通り、テストブランチの strings_template.po を POEdit を使って strings.po に適用し、separate.sh を実行するだけです。

**Reply by denev-rove** [reply]
了解しました。
ご教示ありがとうございます。
私は普段テストバージョンで遊んでいるので、バージョン更新に伴う問題が発生した場合は、分かる範囲で調査して、必要があれば情報提供したいと思います。

いつも更新作業おつかれさまです。
丁度いい機会なので、改めてお礼を申し上げます。
